profiling.cpp causes compilation to fail

profiling.cpp causes compilation to fail on windows (have not tested other operating systems). deleting profiling.cpp allows compilation to succeed.

=====
src\profiling.cpp:3:1: error: 'os_log_t' does not name a type
    3 | os_log_t profiling_os_log = os_log_create("org.godot.gdextension.hexmap",
      | ^~~~~~~~

=====

--- a/src/profiling.cpp
+++ b/src/profiling.cpp
@@ -1,4 +1,8 @@
 #include "profiling.h"
 
+#ifdef __APPLE__
+
 os_log_t profiling_os_log = os_log_create("org.godot.gdextension.hexmap",
         OS_LOG_CATEGORY_POINTS_OF_INTEREST);
+
+#endif // __APPLE__

